sl/symgc.cc

```diff
@@ -108,13 +108,14 @@ bool gcCore(SymHeap &sh, TObjId obj, TObjSet *leakObjs, bool sharedOnly)
 
         if (sh.isAnonStackObj(obj))
```

Comment:
Just a small suggestion: `if (!sh.isAnonStackObj(obj))` because an empty statement looks weird. But it may be as it is now.

Reply:
Good point.  Fixed with 174b0af3450dfbcdddbaa7aeba0996e7109daea5.